dashboard/lib/tasks/seed.rake

```diff
@@ -149,7 +149,7 @@ namespace :seed do
 
   # detect changes to dsldefined level files
   # LevelGroup must be last here so that LevelGroups are seeded after all levels that they can contain
```

Comment:
> `# LevelGroup must be last here so that LevelGroups are seeded after all levels that they can contain`

Does BubbleChoice need to be second-to-last, for the same reason?

Reply:
yes, good catch!